README.md

```diff
@@ -2,36 +2,43 @@
 
 A service for assisted code review, that allows running custom code Analyzers on pull requests.
 
+# SDK
+
+If you are developing an Analyzer, please check [SDK documentation](./sdk/README.md).
+
+It includes a curl-style binary that allows to trigger Analyzers directly, without launching a full lookout server.
+
 # Installation
 
 `go get github.com/src-d/lookout`
 
-## Dependencies
+# Dependencies
 
 The included [`./docker-compose.yml`](./docker-compose.yml) allows to start all dependencies using [Docker Compose](https://docs.docker.com/compose/) 
 
 * [bblfshd](https://github.com/bblfsh/bblfshd), on `localhost:9432`
 * [PostgreSQL](https://www.postgresql.org/), on `localhost:5432` password `example`
 
-Clone the repository, or download [`./docker-compose.yml`](./docker-compose.yml), and run:
+Clone the repository, or download [`./docker-compose.yml`](./docker-compose.yml)
 
-```bash
-docker-compose up
-```
+# Usage
 
+To trigger the analysis on an actual pull request of a GitHub repository you will need [GitHub access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
 
-# Example
+## With Docker
 
-## SDK
+Run:
 
-If you are developing an Analyzer, please check [SDK documentation](./sdk/README.md).
-
-It includes a curl-style binary that allows to trigger Analyzers directly, without launching a full lookout server.
-
-## Server
+```bash
+GITHUB_USER=<user> GITHUB_TOKEN=<token> REPO=<github repository> docker-compose up
```

Comment:
A nitpick, but `github repository` here may be a bit confusing: is it a path, or a name of the repo, or a `user/repo`?  Do you think it might be worth to be specific, to avoid the confusion?